tests/test_account_limits.py

```diff
@@ -75,23 +75,42 @@ def test_get_global_account_usage(self, account, rse_factory, db_session):
         results = account_limit.get_global_account_usage(account=account, rse_expression=f'{rse1}|{rse2}', session=db_session)
         assert len(results) == 1
 
-    def test_local_account_limits(self, account, rse_factory, db_session):
+    def test_local_account_limit_multiple_rses(self, account, rse_factory, db_session):
         """ ACCOUNT_LIMIT (CORE): Set, get and delete local account limits """
         _, rse1_id = rse_factory.make_mock_rse()
         _, rse2_id = rse_factory.make_mock_rse()
+        _, rse3_id = rse_factory.make_mock_rse()
         limit1 = 100
         limit2 = 200
         account_limit.set_local_account_limit(account=account, rse_id=rse1_id, bytes_=limit1, session=db_session)
         account_limit.set_local_account_limit(account=account, rse_id=rse2_id, bytes_=limit2, session=db_session)
-        results = account_limit.get_local_account_limits(account=account, session=db_session)
+        account_limit.set_local_account_limit(account=account, rse_id=rse3_id, bytes_=limit2, session=db_session)
+        results = account_limit.get_local_account_limit(account=account, rse_ids=[rse1_id, rse2_id],session=db_session)
```

Comment:
I don't see a need for a third mock RSE here, we already have two RSEs, so we can test the 'multiple RSE' path - to do this kind of testing you could run
1. get_local_account_limits with just rse1_id, see if it returns 1 result
2. get_local_account_limits with both rse1_id and rse2_id, see if it returns both results